pkg/apps/strategy/support/lifecycle.go

```diff
@@ -175,6 +176,9 @@ func (e *hookExecutor) tagImages(hook *appsinternal.LifecycleHook, rc *corev1.Re
 //   * Working directory
 //   * Resources
 func (e *hookExecutor) executeExecNewPod(hook *appsinternal.LifecycleHook, rc *corev1.ReplicationController, suffix, label string) error {
+	// TODO: This should move to external helper once we are sure there are no replication controller with internal deployment config
+	//       serialized.
+	config, err := appsinternalutil.DecodeDeploymentConfig(rc)
```

Comment:
this is probably the side effect of accidentally removing it in the first commit